frontend/src/assets/lang/app/nl.json

```diff
@@ -281,7 +281,14 @@
         },
         "edit": "Bewerken",
         "cancel": "Annuleer",
-        "save": "Sla op",
+        "save": {
+            "save": "Sla op",
+            "error": {
+                "title": "Ongeldige Opsla Operatie",
+                "detail": "Men kan niet opslaan wanneer geen item is geselecteerd"

```

Comment:
This does not sound right.

Maybey: "U probeert een item op te slaan zonder dit te selecteeren."